pkg/state/filter.go

```diff
@@ -58,9 +58,24 @@ func filterOutDynamicAttributes(iface map[string]interface{}) {
 		return
 	}
 
-	bridge := iface["bridge"].(map[string]interface{})
+	bridgeIface, ok := iface["bridge"]
```

Comment:
How is this `iface`? We just know that it has the type of an empty interface. Maybe `bridgeRaw` would be slightly better?